python/tests/test_account.py

```diff
@@ -1470,22 +1469,12 @@ def test_set_get_group_image(self, acfactory, data, lp):
         assert msg_back.chat == chat
         assert chat.get_profile_image() is None
 
-    def test_accept_sender_contact(self, acfactory, lp):
-        ac1, ac2 = acfactory.get_two_online_accounts()
-        ch = ac1.create_chat_by_contact(ac1.create_contact(ac2.get_config("addr")))
-        ch.send_text("hello")
-        msg = ac2._evtracker.wait_next_messages_changed()
-        assert msg.chat.is_deaddrop()
-        msg.accept_sender_contact()
-        assert not msg.chat.is_deaddrop()
-
```

Comment:
I know other tests also test this, but I would still have kept it so that we do not have to rely on that other tests will still test this in the future